src/lib/workspace/sharing/PublishFromAnalysis.tsx

```diff
@@ -0,0 +1,67 @@
+import { useWdkService } from '@veupathdb/wdk-client/lib/Hooks/WdkServiceHook';
```

Comment:
I guess we don't need this file anymore? (Since the finalized UX doesn't offer "make analysis public" from the `AnalysisPanel`.)

Reply:
Correct! I'll toast it! TY